### ISSUE TITLE
vargant up -- RunTimeError: can't modify frozen String

### ISSUE THREAD

**Issue by edgar323** [p4lang/tutorials#131]
Hi all,

I tried running the "vagrant up" cmd within the directory "tutorials/P4D2_2018_East/vm", but I run into the exception:
/**********************************************************************************************/
There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /Users/.../tutorials/P4D2_2018_East/vm/Vagrantfile
Line number: 0
Message: RuntimeError: can't modify frozen String
/**********************************************************************************************/
I have installed Virtual Box and Vagrant and I made sure that my tutorials clone is up to date.
Also, I am running macOS Sierra.

Does anyone have any thoughts on what could be causing this? (Did I miss some steps?...)

Thanks

**Comment by jnfoster** [comment]
This is a new bug introduced in Vagrant 2.0.3.

As a workaround, please comment out or delete the line
```
Vagrant::DEFAULT_SERVER_URL.replace("https://vagrantcloud.com")
```

**Comment by alanbuxey** [comment]
thanks. saw some discussions about atlas.hashicorp.com and vagrantcloud.com
- that comment out is a fix that appears to work just fine :)

alan

**Comment by jnfoster** [comment]
Fixed by #135
